applications/maplab_node/src/synchronizer.cc

```diff
@@ -12,6 +12,12 @@ DEFINE_double(
     vio_nframe_sync_max_output_frequency_hz, 10.0,
     "Maximum output frequency of the synchronized NFrame structures "
     "from the synchronizer.");
+DEFINE_double(
+    vio_nframe_timestamp_diff_tolerance_, 0.95,
```

Comment:
can you name it such that it is clear that it belongs to the flag above? Maybe: vio_nframe_sync_max_output_frequency_tolerance_ ?